src/main/java/be/seeseemelk/mockbukkit/inventory/ChestStateMock.java

```diff
@@ -0,0 +1,26 @@
+package be.seeseemelk.mockbukkit.inventory;
+
+import be.seeseemelk.mockbukkit.block.BlockStateMock;
+import org.bukkit.block.Chest;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.material.MaterialData;
+
+public class ChestStateMock extends BlockStateMock implements Chest {
+    ChestInventoryMock inventory = new ChestInventoryMock(this, "Baú", 9*3);
```

Comment:
Should be private plus is this "Baú" valid?

Reply:
oh sorry im portuguese i had put chest in portuguese which is baú

Reply:
Fixed